Source/WebInspectorUI/UserInterface/Views/BlackboxSettingsView.css

```diff
@@ -112,3 +113,15 @@
     height: 15px;
     margin-inline-start: 4px;
 }
+
+@media (prefers-color-scheme: dark) {
+    @media (prefers-contrast: more) {
+        .settings-view.blackbox > table > tbody td.url {
+            background-color: hsl(0, 0%, 14%);
```

Comment:
why modify just this property instead of more generally adjusting `--background-color-content` to have a better value in more contrast so that everything using it will benefit?  a big part of the reason we use CSS variables is to avoid having to redeclare the same colors over and over across the UI

also, how did you come up with this value specifically?  for example, in the past we've tried to follow Apple HIG <https://developer.apple.com/design/human-interface-guidelines/color>

ditto below

Reply:
Because they both affect the background color, I will refer to the Apple HIG, I did not know there is one. I referred to the WhatWG guidelines: https://webaim.org/articles/contrast/ and https://webaim.org/resources/contrastchecker/.